frontend/server/src/test/resources/config.properties

```diff
@@ -25,7 +25,6 @@ certificate_file=src/test/resources/certs.pem
 # max_response_size=6553500
 max_request_size=10485760
 # blacklist_env_vars=.*USERNAME.*|.*PASSWORD.*
-# enable_env_vars_config=false
```

Comment:
Why are we removing this?

Reply:
The variable name is incorrect. The correct one is already uncommented and in use in line #29 below.

Reply:
Oh. Ok. Good catch.